editor/actions.js

```diff
@@ -56,11 +56,9 @@ export function editPost( edits ) {
 	};
 }
 
-export function savePost( postId, edits ) {
+export function savePost() {
```

Comment:
An interesting question. Another question is whether we want the "Revert to draft" action to immediately request the update, or wait until an explicit Save/Update action. The latter is the behavior of the current editor, but per the mockup shown at https://github.com/WordPress/gutenberg/pull/945#issuecomment-306862458 , the expectation with a button might be more that it would take effect immediately.